src/elements/content-sidebar/activity-feed/activity-feed/ActiveState.js

```diff
@@ -156,6 +157,14 @@ const ActiveState = ({
                 const isFocused = item === activeFeedItem;
                 const refValue = isFocused ? activeFeedItemRef : undefined;
                 const itemFileVersionId = getProp(item, 'file_version.id');
+                const commentProps = {
```

Comment:
Both BaseComment and Replies have duplicate props in this case, so I created a variable to remove duplication. I named it commentProps since they are props needed by BaseComment